src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (a = null) => {
+    if (a === null) {
```

Comment:
```suggestion
    if (a === undefined) {
```
You can just check if it's undefined in case parameter wasn't passed